src/test/java/com/app/rest/test/service/ItemTest.java

```diff
@@ -0,0 +1,59 @@
+package com.app.rest.test.service;
+
+
+import com.app.rest.model.dto.ItemDTO;
+import com.app.rest.model.persistence.ItemDetail;
+import com.app.rest.service.ItemService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.NoSuchElementException;
+
+@SpringBootTest
+public class ItemTest {
+
+    @Autowired
+    private ItemService itemService;
+
+    @Test
+    public void testItems() {
+        System.out.println(itemService.getItems());
+    }
+
+    @Test
+    public void testItem() {
+        itemService.getItemById(1L);
+        try {
+            itemService.getItemById(8L);
+        } catch (NoSuchElementException nf) {
+            System.out.println("Id 8 not found, which is expected");
```

Comment:
con JUnit se pueden hacer asserts si levanta una Exception y es esperada -> https://stackoverflow.com/questions/156503/how-do-you-assert-that-a-certain-exception-is-thrown-in-junit-4-tests/2935935#2935935

Reply:
Si pero esto no es junit, ni siquiera son test de unidad, son cosas q hice para ir probando q retornaba y que venia, sin nteer q andar usando el controller, no son TESTS SERIOS. No se hacer tests serios.

Reply:
jajajjaa bueno, vamos a ir incorporándolos, suma ir metiendo tests